### PR TITLE
RPE-752 Output vars needed by processor

### PR DESCRIPTION
**Pull request by timja**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RPE-752

### Change description ###
These vars are needed in processor,
also jenkins-library branch is now merged so back to master for it

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: storage-account.tf [hunk 1]
```diff
@@ -54,3 +54,11 @@ resource "azurerm_storage_container" "sscs_container" {
   resource_group_name  = "${azurerm_storage_account.storage_account.resource_group_name}"
   storage_account_name = "${azurerm_storage_account.storage_account.name}"
 }
+
+output "storage_account_name" {
+  value = "${azurerm_storage_account.storage_account.name}"
```
Comment (nitinprabhu) [inline storage-account.tf]: could you please add the sensitive flag as we don't wan't to see them in Jenkins log ?

File: storage-account.tf [hunk 2]
```diff
@@ -54,3 +54,11 @@ resource "azurerm_storage_container" "sscs_container" {
...
+}
+
+output "storage_account_primary_key" {
+  value = "${azurerm_storage_account.storage_account.primary_access_key}"
```
Comment (nitinprabhu) [inline storage-account.tf]: same as above

File: storage-account.tf (hunk 1 above)
Comment (timja) [reply storage-account.tf]: the flag doesn't do anything

File: storage-account.tf (hunk 2 above)
Comment (timja) [reply storage-account.tf]: I've marked this one as sensitive because it is, but it doesn't do anything at all...

Comment (nitinprabhu) [reply storage-account.tf]: it masks the value in the Jenkins log :)

Comment (timja) [reply storage-account.tf]: the pipeline will still show them

**Pull request merged**